tools/PsesLogAnalyzer/PsesLogAnalyzer.psm1

```diff
@@ -0,0 +1,5 @@
+. $PSScriptRoot\Types.ps1
+. $PSScriptRoot\Parse-PsesLog.ps1
+. $PSScriptRoot\Analyze.ps1
+
+Export-ModuleMember -Function *-*
```

Comment:
@daxian-dbw thinks that this should be unneeded with `FunctionsToExport` set in the manifest.

Apparently the flow is:

**defined commands** --> `Export-ModuleMember` --> `FunctionsToExport` --> **actual exported commands**

Where `-->` is "filter through".

Reply:
However, this bodes experimentation

Reply:
Certainly defining an explicit list of exported functions is something we want to encourage for performance and discovery/analysis reasons.

Reply:
Fixed.